python_bot.py

```diff
@@ -567,7 +579,7 @@ def emojify(string):
   elif(message.content.startswith('!pizza')):
     await client.send_message(message.channel, 'Pizza? Who\'s paying for this? Not me.')
   elif(message.content.startswith('!Mugglewump')):
-    if(message.author.id == '159785058381725696' or message.author.id == '83809782691004416'):
+    if(message.author.id == id_branden or message.author.id == '83809782691004416'):
```

Comment:
nit: Same here for this id. Infact, you could just remove this entire `Mugglewump` command if you want. This is someone from an old discord server